LambdaSource/Scotty_TableSlotUpdater/lambdaHandler.py

```diff
@@ -1,10 +1,98 @@
 #! python3
 import boto3
-import argparse
-
+import time
 from pprint import pprint
 
 
+def updateBot(lex, updatedSlotVersion):
+    bot = lex.get_bot(
+        name='Scotty',
+        versionOrAlias="$LATEST"
+    )
+    intentVersion = []
+    intentName = []
+    for intent in bot['intents']:
+        newIntent = lex.get_intent(name=intent['intentName'], version="$LATEST")
+        for i in range(len(newIntent['slots'])):
+            if "table" == newIntent['slots'][i]['name']:
+                newIntent['slots'][i]['slotTypeVersion'] = updatedSlotVersion
+                putIntent(lex, newIntent)
```

Comment:
As a general comment - anytime you are doing something that involves a request (in this case to AWS to update something) you should be checking the return code, and not just assuming success.